src/store/model/euronics-de.ts

```diff
@@ -8,20 +8,52 @@ export const EuronicsDE: Store = {
 		},
 		maxPrice: {
 			container: '.price--content',
-			euroFormat: true
+			euroFormat: false
 		},
 		outOfStock: {
-			container: '.product--buybox .alert--content',
-			text: ['Artikel steht derzeit nicht zur Verfügung']
+			container:
+				'.product--buybox .alert.is--error.is--rounded .alert--content',
+			text: [
+				'Artikel steht derzeit nicht zur Verfügung',
+				'Morgen im Laufe des Morgens und nur online unter'
+			]
 		}
 	},
 	links: [
+		{
+			brand: 'test:brand',
+			model: 'test:model',
+			series: 'test:series',
+			url:
+				'https://www.euronics.de/telefon-und-navigation/festnetz/schnurlose-telefone/kx-tg6721gb-schnurlostelefon-mit-anrufbeantworter-schwarz-4051168442801'
```

Comment:
It seems that this content is in fact in `euroFormat`

`31,16 €`

Reply:
Just for my understanding, as I think I may have misunderstood the use of euroFormat.
Is it to toggle between "." for true and "," for false as separator between the thousands?
If yes, then give me some minutes, I'll fix the mess I did here.

Reply:
Yep! That's exactly it. Thanks for fixing and also updating old ones 😄